pkg/alpine/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM lfedge/eve-alpine-base:f11c7a5dcffe1ab7b466bea7e26010a27da62b3b AS cache
+# image was bootstraped using FROM lfedge/eve-alpine-base:f11c7a5dcffe1ab7b466bea7e26010a27da62b3b AS cache
```

Comment:
Would it make sense to add a comment with a reference to the markdown file here?